src/platform/site-wide/sass/modules/_m-side-nav.scss

```diff
@@ -15,212 +14,223 @@ $side-nav-color-dark: #323a45;
     height: auto;
   }
   .va-sidenav-default-trigger {
-    padding: 0 10px;
-    color: $color-primary;
+    padding: 5px 0 0 22px;
```

Comment:
I know @ncksllvn and I have been trying to avoid adding custom styles in our code lately and instead trying to rely on [design utility classes](https://design.va.gov/utilities/) to achieve the same thing 🙂I think quite a few of these custom styles could be replaced with the design utility classes if you want!

Reply:
I feel you for sure on that - I have a few cases that the design utilities don't _seem to_ cover in this pr. In this case, that specific padding doesn't seem to have the right combination of options, @kelsonic